2-Browsers/Week1/homework/ex4-whatsTheTime/index.js

```diff
@@ -7,6 +7,9 @@
 ------------------------------------------------------------------------------*/
 function addCurrentTime() {
   // TODO complete this function
+  const today = new Date().toLocaleTimeString();
+  document.body.innerHTML = today;
 }
 
 // TODO execute `addCurrentTime` when the browser has completed loading the page
+window.addEventListener('load', setInterval(addCurrentTime, 1000));
```

Comment:
If you want the `setInterval` to happen on window load, you should make sure it's wrapped in a function here:

```javascript
 window.addEventListener('load', () => setInterval(addCurrentTime, 1000));
```

If you don't do this, Javascript will immediately call `setInterval(...)` and set the _result_ of that (the interval identifier) as the second parameter to `addEventListener`.

Reply:
Yes ,
this is great note

I will do it